prober/tls.go

```diff
@@ -69,6 +70,17 @@ func getLastChainExpiry(state *tls.ConnectionState) time.Time {
 	return lastChainExpiry
 }
 
+func getSerialNumber(state *tls.ConnectionState) string {
+	cert := state.PeerCertificates[0]
+	// Actual serial number = 0B:FF:BC:11:F1:90:7D:02:AF:71:9A:FC:D6:4F:B2:53
```

Comment:
let's change the comment to be a little short and easy to read, I was thinking something like this:

```
using `cert.SerialNumber.Text(16)` will drop the leading zeros when converting the SerialNumber to String.
To avoid that, we format in lowercase the bytes with `%x` to base 16, with lower-case letters for a-f.
```

feel free to include link to https://go.dev/play/p/Fylce70N2Zl, and https://github.com/mozilla/tls-observatory/pull/245 (_this issue is cited as source in https://github.com/atc0005/check-cert, so let's cite the same source_)

Reply:
Hi, updated as requested